server/src/database/migrations/sqlite/001-setup.sql

```diff
@@ -70,5 +70,16 @@ CREATE TABLE reactions (
 -- Table: Migrations
 CREATE TABLE migrations (id INTEGER PRIMARY KEY, file VARCHAR(255) UNIQUE);
 
+-- Table: Instance runners
+CREATE TABLE instance_runners (
+  tag       VARCHAR(32) UNIQUE NOT NULL,
+  time     VARCHAR(30) DEFAULT (CURRENT_TIMESTAMP) NOT NULL,
+  name     VARCHAR(32) NOT NULL,
+  user_id  CHAR(8) NOT NULL,
+  size     INT,
```

Comment:
Can this be NULL?

Reply:
yes, on default images

Reply:
and I just realized that default images are not in the db anymore...

Reply:
This was fixed in c216773